queries.sql

```diff
@@ -36,40 +36,41 @@ SELECT * FROM albums WHERE title LIKE 'A%' ORDER BY title;
 -- Remember: run "\d+ invoices" to see the structure of the "invoices" table
 
 -- Every invoice
-
+SELECT * FROM invoices;
 -- Every invoice ordered by total invoice amount ("total")
-
+SELECT * FROM invoices ORDER BY total;
 -- Every invoice with a total greater than 10
-
+SELECT * FROM invoices WHERE total>10;
 -- The 10 least expensive invoices
 -- Remember: ORDER BY orders from lowest-to-highest by default
 SELECT * FROM invoices ORDER BY total LIMIT 10;
 
 -- The 10 most expensive invoices
-
+SELECT * FROM invoices ORDER BY total DESC LIMIT 10;
 -- The 15 most recent invoices
-
+SELECT * FROM invoices ORDER BY invoice_date DESC LIMIT 15;
 -- The 15 oldest invoices
-
+SELECT * FROM invoices ORDER BY invoice_date LIMIT 15;
 -- The 10 most expensive invoices from the US
-
+SELECT * FROM  invoices WHERE billing_country='USA' ORDER BY total DESC LIMIT 10;
 -- The 10 least expensive invoices from the US
-
+SELECT * FROM  invoices WHERE billing_country='USA' ORDER BY total LIMIT 10;
 -- The 10 most expensive invoices from outside the US
 -- Hint: If "=" means equal, use "!=" to mean "not equal"
-
+SELECT * FROM  invoices WHERE billing_country!='USA' ORDER BY total DESC LIMIT 10;
 -- Every invoice from Chicago, IL
 SELECT * FROM invoices WHERE billing_city = 'Chicago' AND billing_state = 'IL' AND billing_country='USA';
 
 -- A list of all the invoices worth more than $5.00 from Chicago, IL
+SELECT * FROM invoices WHERE billing_city = 'Chicago' AND billing_state = 'IL' AND billing_country='USA' AND total>5.00;
 
 -- The billing addresses of the 5 most valuable invoices from Mountain View CA
 -- Gotta reward those big spenders!
-
+SELECT * FROM invoices WHERE billing_city = 'Mountain View' AND billing_state = 'CA' AND billing_country='USA' DESC LIMIT 5;
 -- A list of the 10 most valuable invoices made before January 1st, 2010
 -- Hint: Dates are formatted like 'YYYY-MM-DD' and you can compare them using '<', '>', '<=' and '>='
 
-
+SELECT * FROM invoices WHERE invoice_date < "2010-01-01" ORDER BY invoice_date DESC LIMIT 10;
```

Comment:
Close! In PostgreSQL, there's a big difference between double quotes `"` and single quotes `'`. To refer to a string, you can only use single quotes, so `invoice_date < '2020-01-01'` will work.

In PostgreSQL, double quotes are used if a table name or column contains a space or other non-letter character (which is generally not recommended). When it sees `"2010-01-01"`, it's going to look for a column named `2020-01-01` and then complain because no such column exists.

You'd have no way of knowing this, so don't sweat it.

See:

- https://stackoverflow.com/questions/41396195/what-is-the-difference-between-single-quotes-and-double-quotes-in-postgresql